examples/show.rs

```diff
@@ -152,9 +152,6 @@ fn resize_window(display: &Display, image: &RawImage2d<'static, u8>) {
     if width < 50 && height < 50 {
         width *= 10;
         height *= 10;
-    } else if width < 5 && height < 5 {
-        width *= 10;
-        height *= 10;
     }
```

Comment:
This one was particularly weird as there's no scenario where that else could ever even happen.